geometry_msgs/Transform end_effector_pose
bool tcp_frame_conversion
---

float32 manipulability_index